modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -30,35 +38,57 @@ If you need to modify or increase a specific quota, see Amazon's documentation o
 |ec2
 |L-1216C47A
 |5
-|100
-| Maximum number of vCPUs assigned to the Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances.
-
-The default value of 5 vCPUs is not sufficient to create ROSA clusters. ROSA has a minimum requirement of 100 vCPUs for cluster creation.
-
+a|
+ifndef::openshift-rosa-hcp[]
+100
+endif::[]
+ifdef::openshift-rosa-hcp[]
+32
+endif::[]
+|Maximum number of vCPUs assigned to the Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances. The default value of 5 vCPUs is not sufficient to create ROSA clusters.
+
+//gp2 is not used for HCP clusters
+ifndef::openshift-rosa-hcp[]
 |Storage for General Purpose SSD (gp2) volume storage in TiB
 |ebs
 |L-D18FCD1D
 |50
 |300
-| The maximum aggregated amount of storage, in TiB, that can be provisioned across General Purpose SSD (gp2) volumes in this Region.
+|The maximum aggregated amount of storage, in TiB, that can be provisioned across General Purpose SSD (gp2) volumes in this Region.
```

Comment:
```suggestion
|The maximum aggregated amount of storage, in TiB, that can be provisioned across General Purpose SSD (gp2) volumes in this region.
```

Replace/Region/region unless there is a specific need to use upper case

Reply:
Region is like Operator, alas